### DISCUSSION TITLE
微信警告第三方插件

### DISCUSSION THREAD

**Discussion by feifeirun97** [MustangYM/WeChatExtension-ForMac#1144] [General]
今天提示我不允许使用第三方插件，下次要封号，有人遇到相同情况吗，我已经吧小助手卸载了，心有余悸

**Comment by chenyida1** [comment]
我也是 已经被提示2次了

**Comment by kaiserh0424** [comment]
过去几天被封2次了，使用了双开和防撤回功能……担心永久封号，暂时卸载了小助手。

**Reply by kaiserh0424** [reply]
微信正式宣布新规，7月15日起，只要触碰三条红线，将被直接封号：

https://www.163.com/dy/article/GEATQJ3R0532P501.html

**Comment by renguagua** [comment]
已经卸载了，还有没有其他的不会被微信检测到的插件啊
